src/tablet/tablet_impl_test.cc

```diff
@@ -5905,13 +5905,6 @@ TEST_F(TabletImplTest, DelRecycleMem) {
     ::openmldb::base::GetChildFileName(FLAGS_recycle_bin_root_path, file_vec);
     ASSERT_EQ(4, (signed)file_vec.size());
 
-    std::cout << "sleep for 35s" << std::endl;
-    sleep(35);
-
-    file_vec.clear();
-    ::openmldb::base::GetChildFileName(FLAGS_recycle_bin_root_path, file_vec);
-    ASSERT_EQ(2, (signed)file_vec.size());
```

Comment:
after dynamic link ,this assert will fail, no idea why .

Reply:
https://stackoverflow.com/questions/60823279/why-memset-is-needed-before-strptime